### PR TITLE
Aizad/ Update scss file structure

### PR DESCRIPTION
**Pull request by aizad-deriv**
- Restructuring the current `SCSS` files following industry practices.
- Removing unused styles and libraries
- Re-adding all the root values and colors following current design implementation.
- Supporting both light and dark color scheme based on current design implementation.
- One source of Truth for all styles definitions

### CODE REVIEW

File: lib/styles/wallet/styles/devices.scss [hunk 1]
```diff
@@ -1,6 +1,3 @@
-/**
- * Define Breakpoints here.
- */
 $tablet-width: 768px;
 $desktop-width: 1024px;
```
Comment (shayan-deriv) [inline lib/styles/wallet/styles/devices.scss:2]: shouldn't we update this with the latest values confirmed by the designers? 
```
600px; // max-width for mobile
1280px; // min-width for desktop
// for tablet the range is in between
```

File: lib/styles/abstracts/_devices.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/**
+ * Define Breakpoints here.
+ */
+$mobile-width: 320px;
+$tablet-width: 768px;
+$desktop-width: 1024px;
```
Comment (shayan-deriv) [inline lib/styles/abstracts/_devices.scss:6]: the same here

File: lib/styles/wallet/styles/devices.scss (hunk 1 above)
Comment (aizad-deriv) [reply lib/styles/wallet/styles/devices.scss:2]: Will do boss 👍🏼

Comment (shayan-deriv) [reply lib/styles/wallet/styles/devices.scss:2]: so for here we don't need any changes? 🤔

Comment (shayan-deriv) [reply lib/styles/wallet/styles/devices.scss:2]: if not let me know to merge it 💯

**Review by wojciech-deriv — APPROVED** [review]

**Pull request merged**